src/components/ButtonModal/ButtonModal.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useAppDispatch } from '../../app/hooks';
+import { actions as currTodoActions } from '../../features/currentTodo';
+
+export const ButtonModal: React.FC = () => {
+  const dispatch = useAppDispatch();
+
+  return (
+    // eslint-disable-next-line jsx-a11y/control-has-associated-label
+    <button
+      type="button"
+      className="delete"
+      data-cy="modal-close"
+      onClick={() => dispatch(currTodoActions.removeTodo())}
```

Comment:
I think it is too long/complicated method for writing it inline, move it into certain function `handleDeleteTodo `for example